includes/Checker/CLI_Runner.php

```diff
@@ -48,24 +48,25 @@ public function is_plugin_check() {
 	}
 
 	/**
-	 * Creates and returns an instance of the Checks class based on the request.
+	 * Returns the plugin parameter based on the request.
 	 *
 	 * @since n.e.x.t
 	 *
-	 * @return Checks An instance of the Checks class.
+	 * @return string The plugin slug.
 	 *
-	 * @throws Exception Thrown if the plugin main file cannot be found based on the CLI input.
+	 * @throws Exception Thrown if the plugin slug is invalid.
 	 */
-	protected function get_checks_instance() {
-		if ( ! isset( $this->checks ) ) {
-			// Get the plugin name from the command line arguments.
-			$plugin_slug = isset( $_SERVER['argv'][3] ) ? $_SERVER['argv'][3] : '';
-			$plugin_file = Plugin_Request_Utility::get_plugin_basename_from_input( $plugin_slug );
+	protected function get_plugin_param() {
+		// Get the plugin name from the command line arguments.
+		$plugin_slug = isset( $_SERVER['argv'][3] ) ? $_SERVER['argv'][3] : '';
 
-			$this->checks = new Checks( WP_PLUGIN_DIR . '/' . $plugin_file );
+		if ( empty( $plugin_slug ) ) {
+			throw new Exception(
+				__( 'Invalid plugin slug: Plugin slug must not be empty.', 'plugin-check' )
+			);
 		}
 
-		return $this->checks;
+		return $plugin_slug;
```

Comment:
See above regarding naming of this. Also can you similarly adjust the exception message to refer to "plugin parameter" rather than "plugin slug"?